cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,553 @@
+General information on rule format
+==================================
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+File Structure
+--------------
+
+Enrichment rules are defined in YAML. The file extension has to be ".yml". Indentation is mandatory within the
+file and has to follow the YAML format strictly. To use a rule it has to be imported in the PIM.
+
+This file starts with "rules" root element, which contains the list of enrichment rules. This document is about this
+list. Each rule is referred to by a code and can contain a list of conditions or actions. Actions are executed if all
+conditions are verified or if no action is defined.
+
+.. code-block:: yaml
+
+    # Example of a file with 2 rules
+    rules:
+        camera_set_canon_brand:
+            priority: 0
+            conditions:
+                - field: family.code
+                operator: IN
+                value:
+                    - camcorders
+                - field: name
+                operator: CONTAINS
+                value: Canon
+            actions:
+                - type: set_value
+                field: camera_brand
+                value: canon_brand
+        camera_copy_name_to_model:
+            priority: 0
+            conditions:
+                - field: family.code
+                operator: IN
+                value:
+                    - camcorders
+            actions:
+                - type: copy_value
+                from_field: name
+                to_field: camera_model_name
+
```

Comment:
Add : 

For a rule without conditions, set the following condition:

[free rule code]:
	priority:
        conditions:[]
        actions:
             - type: